data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping

```diff
@@ -1,8 +1,29 @@
 CLASS net/minecraft/world/entity/ai/behavior/AcquirePoi
+	METHOD <init> (Lnet/minecraft/world/entity/ai/village/poi/PoiType;Lnet/minecraft/world/entity/ai/memory/MemoryModuleType;Lnet/minecraft/world/entity/ai/memory/MemoryModuleType;ZLjava/util/Optional;)V
+		ARG 1 poiType
+		ARG 3 memoryToAcquire
+		ARG 4 onlyIfAdult
+		ARG 5 onPoiAcquistutionEvent
+	METHOD <init> (Lnet/minecraft/world/entity/ai/village/poi/PoiType;Lnet/minecraft/world/entity/ai/memory/MemoryModuleType;ZLjava/util/Optional;)V
+		ARG 1 poiType
+		ARG 3 onlyIfAdult
+		ARG 4 onPoiAcquisitionEvent
 	METHOD checkExtraStartConditions (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/entity/PathfinderMob;)Z
 		ARG 1 level
 		ARG 2 owner
+	METHOD constructEntryConditionMap (Lnet/minecraft/world/entity/ai/memory/MemoryModuleType;Lnet/minecraft/world/entity/ai/memory/MemoryModuleType;)Lcom/google/common/collect/ImmutableMap;
+		ARG 1 memoryToAcquire
```

Comment:
The order is:
``memoryKey`` & ``memoryToAcquire`` i think.
Either way the second value is ``memoryToAcquire`` not the first, see the call to super in AcquirePoi constructor.

Reply:
I'll add `memoryKey` as the name for the first parameter, but in Enigma it already shows `memoryToAcquire` as the second parameter, not the first.